css/style.css

```diff
@@ -0,0 +1,66 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  color: black;
+  font-weight: bold;
+}
+
+.container {
+  width: 70%;
+  margin: auto;
+}
+
+.book-title {
+  margin: 5% 0;
+  text-align: center;
+}
+
+.book-list {
+  display: grid;
+  grid-template-columns: 6fr 1fr;
+  row-gap: 10px;
+  border: 5px solid #000;
+  padding: 10px;
+  margin-bottom: 5%;
+}
```

Comment:
- [ ] Kindly add the extra background color to your booklist so as to follow the wire design.
- Your design:
![Annotation 2022-04-19 223931](https://user-images.githubusercontent.com/85113567/164107044-1063fedd-b8c7-48f6-b3fe-bfe190488f86.jpg)
- Wireframe:
![Annotation 2022-04-19 224010](https://user-images.githubusercontent.com/85113567/164107085-a1539f9e-1cf9-4ca2-a4c4-f5ccff2d61b3.jpg)